### PR TITLE
Add facepile variant template to peoplepicker component

### PR DESCRIPTION
**Pull request by philworthington**
- Add separate membersList variant file
- Clean up and update membersList markup
- Update img tags used in peoplepicker when a result is selected

### CODE REVIEW

**Comment by mikewheaton** [comment]
Looks like there are some JS issues on the component sample page. Showed it to @philworthington and he's going to work on a fix.

**Comment by philworthington** [comment]
js issue fixed

**Comment by mikewheaton** [comment]
Looks good! Can we get one more reviewer to take a look?

File: src/components/PeoplePicker/Jquery.PeoplePicker.js [hunk 1]
```diff
@@ -290,7 +297,7 @@ var fabric = fabric || {};
         /** Show members and hide searchmore when field is empty */
         if ($(this).val() === "") {
           $peoplePicker.removeClass('is-searching');
-          $selected.show();
+          // $selected.show();
```
Comment (Jahnp) [inline src/components/PeoplePicker/Jquery.PeoplePicker.js]: Please remove rather than leave commented-out.

**Comment by Jahnp** [comment]
Looks solid for the most part--there's just one small nit. Feel free to merge when that's been addressed!

**Pull request merged**